dev/sparktestsupport/modules.py

```diff
@@ -612,6 +612,9 @@ def __hash__(self):
         "pyspark.pandas.typedef.typehints",
         # unittests
         "pyspark.pandas.tests.test_dataframe",
+        "pyspark.pandas.tests.test_ops_on_diff_frames_groupby_rolling",
```

Comment:
We should add `pyspark.pandas.tests.test_ops_on_diff_frames` as well?

Reply:
Good catch! Added it.